01-intro-to-R.Rmd

```diff
@@ -247,10 +249,8 @@ doing.
 ### Vectors and data types
 ```
 
-A vector is the most common and basic data structure in R, and is pretty much
-the workhorse of R. It's a group of values, mainly either numbers or
-characters. You can assign this list of values to a variable, just like you
-would for one item. For example we can create a vector of animal weights:
+A vector is the most common and basic data type in R, and is pretty much
```

Comment:
Maybe: 
"We can assign a series of values to a vector using the `c()` function, with the values to combine in the vector as its argument." --> We can assign a series of values to a vector using the `c()` function, with the values to combine in the vector comma-separated within the parentheses.